README.rst

```diff
@@ -26,6 +26,11 @@ Installing
     pip install boxsdk
 
 
+
+.. contents:: :depth: 1
```

Comment:
This is a strange spot for a toc - shouldn't it come before the first section (installing)?